framework/src/Volo.Abp.BlobStoring.Azure/Volo.Abp.BlobStoring.Azure.csproj

```diff
@@ -16,7 +16,7 @@
 
     <ItemGroup>
         <ProjectReference Include="..\Volo.Abp.BlobStoring\Volo.Abp.BlobStoring.csproj" />
-        <PackageReference Include="Azure.Storage.Blobs" Version="12.12.0" />
+        <PackageReference Include="Azure.Storage.Blobs" Version="12.13.0" />
```

Comment:
Bump Azure.Storage.Blobs from 12.12.0 to 12.13.0 in /framework/src/Volo.Abp.BlobStoring.Azure #4